packages/cursorless-cheatsheet/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@cursorless/cursorless-cheatsheet",
+  "version": "0.1.0",
+  "description": "Cursorless cheatsheet implementation",
+  "type": "module",
+  "main": "./out/index.js",
+  "types": "./out/index.d.ts",
+  "scripts": {
+    "compile:tsc": "tsc --build",
+    "compile:esbuild": "esbuild ./src/index.ts --sourcemap --format=esm --bundle --packages=external --outfile=./out/index.js",
+    "compile": "pnpm compile:tsc && pnpm compile:esbuild",
+    "watch:tsc": "pnpm compile:tsc --watch",
+    "watch:esbuild": "pnpm compile:esbuild --watch",
+    "watch": "pnpm run --filter @cursorless/cursorless-cheatsheet --parallel '/^watch:.*/'",
+    "clean": "rm -rf ./out tsconfig.tsbuildinfo ./dist ./build"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "MIT",
+  "exports": {
+    ".": {
+      "cursorless:bundler": "./src/index.ts",
+      "default": "./out/index.js"
+    }
+  },
+  "dependencies": {
+    "@cursorless/common": "workspace:*",
+    "immer": "^10.0.4",
+    "lodash": "^4.17.21",
```

Comment:
don't forget to update this to lodash-es now that #2476 has merged

Reply:
done